# request

---
# response
int32[] ids
geometry_msgs/Point[] points